scheduler/src/cook/rest/api.clj

```diff
@@ -1438,7 +1429,7 @@
   (let [jobs (wrap-seq (get-in ctx [:request :query-params :job]))
         instances (wrap-seq (get-in ctx [:request :query-params :instance]))
         allow-partial-results (get-in ctx [:request :query-params :partial])
-        instance-uuid->job-uuid #(instance-uuid->job-uuid (d/db conn) %)
+        instance-uuid->job-uuid #(util/instance-uuid->job-uuid (d/db conn) %)
```

Comment:
Would it make sense to use the cached call here?

Reply:
In short, yes it can be.

My only concern is that currently we cache the `instance-uuid-->job-uuid` information right before the task is pumped into the k8s state machine, which I think is after that information is added to datomic (I'm not 100% sure). So if this `rest.api` function is called in that interim period before adding `instance-uuid` to the cache but after adding it to the DB, our cache would miss. I'm not super familiar with how often this `retrieve-jobs` api call is generally used, but I assume that it isn't used frequently enough in that interim period to give our cache a significantly increased miss rate.

Reply:
On the subject of this time difference-- Could you point me to where the `instance-uuid-->job-uuid` information is added to datomic? I thought it might be in https://github.com/twosigma/Cook/blob/master/scheduler/src/cook/scheduler/scheduler.clj#L773 where `:job/_instance` is in the map but I wasn't 100% sure if that's what's happening in that function (not too familiar with how datomic+clojure works). I'm interested  to see how early it occurs vs. the time when we cache it.